sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Projection.scala

```diff
@@ -22,9 +22,11 @@ package org.apache.spark.sql.catalyst.expressions
  * @param expressions a sequence of expressions that determine the value of each column of the
  *                    output row.
  */
-class InterpretedProjection(expressions: Seq[Expression]) extends Projection {
-  def this(expressions: Seq[Expression], inputSchema: Seq[Attribute]) =
-    this(expressions.map(BindReferences.bindReference(_, inputSchema)))
+class InterpretedProjection(expressions: Seq[Expression], mutableRow: Boolean = false)
+  extends Projection {
+  def this(expressions: Seq[Expression],
+           inputSchema: Seq[Attribute], mutableRow: Boolean = false) =
```

Comment:
Sorry for late reply. It happened when I changed empty input in AggregateSuite test to something like ("Hello", 4, 2.0). In GeneratedAggreate, whenever the grouping expression is empty or unsafeEnabled is not applied for some reason spark tries to case GenericRow to MutableRow.